app/src/main/res/xml/backup_descriptor.xml

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<full-backup-content>
-    <!-- Exclude specific shared preferences that contain GCM registration Id -->
-</full-backup-content>
```

Comment:
Why did you delete this file ? Is it really useless ?

Reply:
Well I followed Cirrus' advice, it was not used anywhere and built was done without any issues.